.github/workflows/stale_issue.yml

```diff
@@ -27,12 +27,12 @@ jobs:
 
         # Don't set closed-for-staleness label to skip closing very old issues
         # regardless of label
-        closed-for-staleness-label: closed-for-staleness
+        # closed-for-staleness-label: closed-for-staleness
```

Comment:
Do we want to comment out this? This is not commented out in https://github.com/awslabs/aws-c-s3/pull/291/files, so I am a bit confused. We should make them consistent.

Reply:
Fixed. Thanks for catching this